lib/scrape.js

```diff
@@ -127,6 +128,18 @@ async function fetchDates() {
     (a, b) => b.completed_task_instance_count - a.completed_task_instance_count
   )
 
-  fs.writeFileSync(`${__dirname}/../out/data.json`, JSON.stringify(data))
+  // readable JSON
+  fs.writeFileSync(
+    `${__dirname}/../out/data.json`,
+    JSON.stringify(data, null, 2)
+  )
+  // minified JSON
+  fs.writeFileSync(
+    `${__dirname}/../out/data.min.json`,
+    JSON.stringify(data)
+  )
```

Comment:
ESLint wants this all on one line

Reply:
i don't know why @gitmate-bot is not screaming at this

@jayvdb ^

Reply:
no idea

Reply:
Shit, it might be missing the eslint plugin.